docs/modules/ROOT/pages/references/functional-requirements.adoc

```diff
@@ -0,0 +1,91 @@
+= Functional Requirements
+
+This page discusses functional requirements.
+It currently mainly targets the Control API and the {product} Portal.
+
+== Feature: Authentication
+
+> As a xref:references/glossary.adoc#_user[User] +
+> I want to only login once to the {product} ecosystem +
+> So that I can use strong authentication methods and still get a good user experience
+
+*Scenarios*
+
+* Login
+* Logout
+
+== Feature: Manage Organizations
+
+> As a xref:references/glossary.adoc#_user[User] +
+> I want to manage organizations +
+> So that I can group resources under it
+
+*Scenarios*
+
+* Create Organization
+* List Organizations
+* Edit Organization
+* Delete Organization
+* Invite User to Organization
+* Join Organization
+
+== Feature: Manage Teams
+
+> As an xref:references/glossary.adoc#_organization-owner[Organization Owner] +
+> I want to manage teams in an organization +
+> So that I can group users under it for easier permission management
+
+*Scenarios*
+
+* Create Team
+* List Teams
+* Edit Team
+* Delete Team
+* Add User to Team
+* Remove User from Team
+
+== Feature: Zones
+
+> As a xref:references/glossary.adoc#_user[User] +
+> I want to see all available {zone}s +
+> So that I know of them
```

Comment:
```suggestion
> As a xref:references/glossary.adoc#_user[User] +
> I want to see a list of all available {zone}s +
> So that I know the zone details
```